plugins/headings/index.js

```diff
@@ -60,11 +60,35 @@ const ERRORS = {
 
         return {
             title: `
-                Nonconsecutive heading level used (h${prevLevel} &rarr;
+                Error: Nonconsecutive heading level used (h${prevLevel} &rarr;
                 h${currLevel})
             `,
             description: description
         };
+    },
+
+    HIDDEN_HEADER(level) {
+        return {
+            title: `Warning: Hidden heading level (h${level})`,
+            description: `
+                <div>
+                    This document contains a hidden h${level}
+                </div>
+            `,
+            errorLevel: "warning",
+        };
+    },
+
+    ARIA_HIDDEN_HEADER(level) {
+        return {
+            title: `Warning: ARIA Hidden heading level (h${level})`,
+            description: `
+                <div>
+                    This document contains an h${level} with aria-hidden set to true.
+                </div>
+            `,
+            errorLevel: "warning"
+        };
```

Comment:
Having the idea of warnings for this makes sense. I mean, hidden headers could be a mistake or it could be intentional.

I wonder if we should add a feature to control what level of issue we annotate? Like a cumulative level thing that says "show all annotations of level X and above"? What do you think? (not suggesting you do that as part of this PR, just thinking aloud).

Reply:
I think that definitely makes sense if you introduce levels.

It could work sort of like the Console log-level filtering in your browser's Developer Tools.

It might even make sense to have each plugin be able to define what its defaults are and fall back to some global default of `["error", "warning"]` if no levels are specified by the plugin.